src/klein/resource.py

```diff
@@ -278,3 +278,97 @@ def processing_failed(failure, error_handlers):
         d.addErrback(processing_failed, self._app._error_handlers)
         d.addCallback(write_response).addErrback(log.err, _why="Unhandled Error writing response")
         return server.NOT_DONE_YET
+
+class Arguments(object):
+    """
+    A wrapper around the C{dict} type that gives easy access to the
+    C{bytes} keys set by Twisted.
+    """
+
+    def __init__(self, args=None):
+        self.arguments = {} if not args else args
+
```

Comment:
Does this attribute need to be public?  If not, let's prefix it with `_` and if it does, it should be documented with @ivar in the class and I think having an attribute with the same name as the class is odd, as that implies they are the same, but why wrap a thing with a same thing?